sktime/forecasting/compose/_fallback.py

```diff
@@ -36,6 +48,22 @@ class is useful in scenarios where the reliability of individual forecasting mod
     verbose : bool, default=False
         If True, raises warnings when a forecaster fails to fit or predict.
 
+    nan_predict_policy: str, default='ignore'
+        Determines the action to take if NaN values are found in the predictions.
+        Available options:
+        * "ignore"
```

Comment:
I think this does not render right if there is no space before and after the bullet point list